search/search_repository.go

```diff
@@ -572,6 +572,7 @@ func (r *GormSearchRepository) SearchFullText(ctx context.Context, rawSearchStri
 	}
 
 	sqlSearchQueryParameter := generateSQLSearchInfo(parsedSearchDict)
+	log.Warn(ctx, map[string]interface{}{"search query": sqlSearchQueryParameter}, "searching for work items")
```

Comment:
Warn?

Reply:
yes, so I have colors in my console when working on it :) Obviously, this was not intended to be part of the PR...